src/console_node/httpUtils.go

```diff
@@ -102,7 +102,7 @@ func postURL(URL string, requestBody []byte, requestHeaders map[string]string) (
 // Helper function to execute an http command
 func getURL(URL string, requestHeaders map[string]string) ([]byte, int, error) {
 	var err error = nil
-	log.Printf("getURL URL: %s\n", URL)
+	//log.Printf("getURL URL: %s\n", URL)
```

Comment:
Same question -- should we just remove this? Or put it behind some conditional, if you're keeping it here for future debugging?

Reply:
Ibid.